torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -2431,7 +2440,11 @@ c10::intrusive_ptr<Work> ProcessGroupNCCL::collective(
   errorIfCapturingNonCapturableNCCL(capture_status);
 
   // Bump collective counter
-  seq_++;
+  if (isP2POp(opType)) {
+    seqP2P_++;
```

Comment:
is this path actually possible?  (to have a p2p op routed into the `ProcessGroupNCCL::collective` function)

Reply:
Not possible! Good catch. Fixed.